.travis.yml

```diff
@@ -28,57 +28,61 @@ dist: trusty
-      - g++-4.8 
-      - gcc-4.8 
-      - git 
-      - git-core 
-      - libasound2-dev 
-      - libcups2-dev 
-      - libelf-dev 
-      - libfreetype6-dev 
-      - libnuma-dev 
-      - libx11-dev 
-      - libxext-dev 
-      - libxrender-dev 
-      - libxt-dev 
-      - libxtst-dev 
-      - make 
-      - openjdk-8-jdk 
-      - pkg-config 
-      - realpath 
-      - ssh 
-      - unzip 
-      - wget 
-      - zip 
+      - autoconf
+      - ca-certificates
+      - ccache
+      - cpio
+      - file
+      - g++-4.8
+      - gcc-4.8
+      - git
+      - git-core
+      - libasound2-dev
+      - libcups2-dev
+      - libelf-dev
+      - libfreetype6-dev
+      - libnuma-dev
+      - libx11-dev
+      - libxext-dev
+      - libxrender-dev
+      - libxt-dev
+      - libxtst-dev
+      - make
+      - openjdk-8-jdk
+      - pkg-config
+      - realpath
+      - ssh
+      - unzip
+      - wget
+      - zip
 before_install:
   - jdk_switcher use oraclejdk8
 env:
   global:
 before_script:
-        #  - ccache -s -z
+  # - ccache -s -z
+  # Exit immediately if any unexpected error occurs.
+  - set -e
   - wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
   - tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
   - cd ..
-  # Shallow clone of the openj9-openjdk-jdk9 repo to speed up clone / reduce server load
+  # Shallow clone of the openj9-openjdk-jdk9 repo to speed up clone / reduce server load.
   - git clone --depth 1 https://github.com/ibmruntimes/openj9-openjdk-jdk9.git
 script:
-  # Clear this option so it doesn't interfere with configure detecting the bootjdk
+  # Clear this option so it doesn't interfere with configure detecting the bootjdk.
   - unset _JAVA_OPTIONS
-  # Point the get_sources script at the OpenJ9 repo that's already been cloned to disk.  
-  # Results in a copy of the source (disk space =( ) but no new network activity so overall a win. 
-  - cd openj9-openjdk-jdk9 && bash ./get_source.sh -openj9-repo=$TRAVIS_BUILD_DIR -openj9-branch=$TRAVIS_BRANCH -openj9-sha=$TRAVIS_COMMIT
-  # Based on https://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/ travis container builds
-  # have 2 cores and 4 gigs of memory.  Attempt to double provision the number of cores for the make...
-  # Limit number of jobs to work around g++ internal compiler error
-  - export UMA_WINDOWS_PARRALLEL_HACK="-j4"
-  - bash ./configure --with-freemarker-jar=$TRAVIS_BUILD_DIR/freemarker.jar --with-jobs=4 --with-num-cores=4
+  # Exit immediately if any unexpected error occurs.
+  - set -e
+  # Point the get_sources script at the OpenJ9 repo that's already been cloned to disk.
+  # Results in a copy of the source (disk space =( ) but no new network activity so overall a win.
+  - cd openj9-openjdk-jdk9 && bash get_source.sh -openj9-repo=$TRAVIS_BUILD_DIR -openj9-branch=$TRAVIS_BRANCH -openj9-sha=$TRAVIS_COMMIT
```

Comment:
We should stop specifying `-openj9-branch` since we really only care about `-openj9-sha`

Reply:
I suppose it doesn't really matter. `$TRAVIS_BRANCH=master` and the default is `master` so it doesn't really change anything. If at some point the get_source script changes its default, we would probably want `TRAVIS_BRANCH` instead, although it doesn't really matter because he checkout the SHA anyways.